src/intervals/trigonometric.jl

```diff
@@ -19,9 +31,17 @@ This is a rather indirect way to determine if π/2 and 3π/2 are contained
 in the interval; cf. the formula for sine of an interval in
 Tucker, *Validated Numerics*."""
 
-function find_quadrants(x::T) where {T<:AbstractFloat}
+function find_quadrants(x::T) where {T}
     temp = atomic(Interval{T}, x) / half_pi(x)
-    (floor(temp.lo), floor(temp.hi))
+
+    return SVector(floor(temp.lo), floor(temp.hi))
```

Comment:
What about returning a tuple here, instead of a `SVector`? (Same comment applies in line 44.)

Reply:
It used to be a tuple; changing it to an `SVector` had a ridiculously big impact on the speed due to the `minimum` and `maximum` below that are not specialised for tuples!

In any case all of this code can be simplified a lot when #119 is possible, I believe.

Reply:
Fair enough.

Reply:
OK, I found an alternative solution: return a tuple and do

`max(tuple...)`

Too late...